phpcs-sniffs/Formidable/Sniffs/WhiteSpace/ConsecutiveAssignmentSpacingSniff.php

```
@@ -0,0 +1,558 @@
+
+		if ( false !== $next && $tokens[ $next ]['code'] === T_OPEN_SQUARE_BRACKET ) {
+			// Include the array access in the signature.
+			$closeBracket = $tokens[ $next ]['bracket_closer'];
+
+			for ( $i = $next; $i <= $closeBracket; $i++ ) {
+				$signature .= $tokens[ $i ]['content'];
+			}
+		}
+
+		return $signature;
+	}
```

PHPMD (2.15.0)

Avoid unused private methods such as 'getVariableSignature'. (undefined)

(UnusedPrivateMethod)